### PR TITLE
Batch update

### LINKED ISSUE #897: Add 'BCHSticker.com' to the 'Retail' category
Requesting to add 'BCHSticker.com' to the 'Retail' category. 

Details follow: 
```yml 
- name: BCHSticker.com
      url: http://BCHSticker.com
      img: 
      twitter: https://yours.org/bchsticker
      email_address: contact@bchsticker.com
      bch: Yes
      btc: No
      othercrypto: No
      doc: https://www.yours.org/content/excited-to-announce-the-launch-of-bchstickercom--dc5394649d16
```

Resources for adding this merchant:
Logo Provided:
BCHSticker.com

[Link to BCHSticker.com](http://BCHSticker.com)
If needed, try the twitter handles profile image: [twitter.com/https://yours.org/bchsticker](https://twitter.com/https://yours.org/bchsticker)

- Verify site is legitimate and safe to list.
- Correct data in form if any is innacurate.

If everything looks okay, Add it to the site:
- Assign to yourself when you begin work.
- Download and resize the image and put it into the proper img folder.
- Add listing alphabetically to proper .yml file.
- Commit changes mentioning this issue number with 'closes #[ISSUE NUMBER HERE]'.

### PR DESCRIPTION
**Pull request by EchterAgo**
closes #897

### CODE REVIEW

**Comment by EchterAgo** [comment]
@kenman345  I fixed it already. I think I'm done for now.

**Comment by EchterAgo** [comment]
Forgot to remove the label.

File: _data/tobacco.yml [hunk 1]
```diff
@@ -76,6 +76,18 @@ websites:
       othercrypto: yes
       doc: https://www.reddit.com/r/btc/comments/7jq1hc/we_just_added_bitcoin_cash_as_a_payment_method_on/
 
+    - name: Liquido24
+      url: https://www.liquido24.de/
```
Comment (kenman345) [inline _data/tobacco.yml]: Can you add the `lang` Tag for these .de sites if they’re primarily for that language. Also, if they’re regional add the region tag and country tags/city tags where pissible

**Comment by kenman345** [comment]
For the list you have remaining just open an issue and paste the list if you wanted help with it but if they’re in something other than English I think most of us won’t be able to help anyways

**Comment by EchterAgo** [comment]
I added lang: de to Bier Kaufen, Avocadostore, Liquido24, Fattoria Sarda and NetflixGutschein. The rest of the German sites are also available in English.

**Comment by kenman345** [comment]
Did you push the commit with the language update?

**Comment by EchterAgo** [comment]
Yes

**Comment by kenman345** [comment]
Oh I realized you were saying that you didn’t need to add it to some of those. Just go ahead and merge then.

**Pull request merged**